jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyWebAppContext.java

```diff
@@ -151,7 +157,7 @@ public void setWebInfIncludeJarPattern(String pattern)
         _webInfIncludeJarPattern = pattern;
     }
 
-    public List<File> getClassPathFiles()
+    public List<Resource> getClassPathFiles()
```

Comment:
There are a huge number of changes in this PR. And a lot of changes to method signatures, classes added and classes deleted. There are too many changes to do this work against jetty-9.4.x, so you'll have to base it off jetty-10.0.x instead.